pytest/lib/cluster.py

```diff
@@ -169,16 +169,18 @@ def rpc_addr(self):
 
 
 class LocalNode(BaseNode):
-    def __init__(self, port, rpc_port, near_root, node_dir, blacklist):
+    def __init__(self, port, rpc_port, near_root, node_dir, blacklist, binary_name='near'):
         super(LocalNode, self).__init__()
         self.port = port
         self.rpc_port = rpc_port
         self.near_root = near_root
         self.node_dir = node_dir
+        self.binary_name = binary_name
+        self.cleaned = False
         with open(os.path.join(node_dir, "config.json")) as f:
             config_json = json.loads(f.read())
-        assert config_json['network']['addr'] == '0.0.0.0:24567', config_json['network']['addr']
-        assert config_json['rpc']['addr'] == '0.0.0.0:3030', config_json['rpc']['addr']
+        # assert config_json['network']['addr'] == '0.0.0.0:24567', config_json['network']['addr']
```

Comment:
Why is this commend out? Should it be removed?

Reply:
yeah, it should be. It's a weird check blocking from running twice node from the same folder.